src/utils/constants/channelLabels.ts

```diff
@@ -0,0 +1,7 @@
+export const channelLabels = {
+  GREY: ['Grey'],
+  GREYA: ['Grey', 'Alpha'],
+  RGB: ['Red', 'Green', 'Blue'],
+  RGBA: ['Red', 'Green', 'Blue', 'Alpha'],
+  BINARY: ['Mask'],
+} as const;
```

Comment:
```suggestion
} as const satisfies Record<ImageColorModel, string[]>;
```

This should make sure we don't forget to update the object if we add new color models.